### ISSUE TITLE
Split Button - Ensure custom controls are keyboard accessible - (2036569848)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5568]
### Module: 
 09b Split Button 

### Violation: 
 Ensure custom controls are keyboard accessible 

![image](https://user-images.githubusercontent.com/5023024/202800066-3874a988-63b3-4fdd-9928-e44cd331706f.png)

### WCAG Reference: 
- [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) 

### Instance ID: 
 2036569848 

### Severity: 
 8 

### Description: 
 [Issue]
There are interactive controls that cannot be navigated to and/or operated with the keyboard alone while a screen reader is running. Examples include:

- All Menu buttons 

[User Impact]
Screen reader users will be prevented from accessing the functionality provided by these controls.

[Code Reference] 
 ```
<button aria-label="Solid Blue Actions" class="icon-end-empty" type="button"><calcite-icon class="icon icon--start" aria-hidden="true" icon="chevronDown" scale="s" calcite-hydrated=""></calcite-icon></button>
 ``` 

### Note: 
 [Suggestion]
Ensure custom controls are keyboard accessible. Users must be able to navigate to and operate controls while a screen reader is running.

[Compliant Code Example] 
 ```
N/A
 ```
 

### Media Type: 
 Custom Controls 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Verified in JAWS and Chrome. Unable to select the split-button dropdown when actively using JAWS.

**Comment by geospatialem** [comment]
Fix will need to include a keydown listener, `onKeyDown` to support `spacebar` and `Enter` keys. Should also accommodate additional keyboard actions for closing. 

The `dropdown` component has a [private function](https://github.com/Esri/calcite-components/blob/master/src/components/dropdown/dropdown.tsx#L545-L576), which we may be able to leverage in this case.

**Comment by geospatialem** [comment]
Reallocating to the April milestone. This should be tackled early on if at all possible, as it effects all keyboard interactions. cc @Elijbet

**Comment by Elijbet** [comment]
> Reallocating to the April milestone. This should be tackled early on if at all possible, as it effects all keyboard interactions. cc @Elijbet

Brief context on this issue. Initially, we tried to address the issue with a VM but had technical difficulties. Later I had a tablet sent to me, but it was not configured correctly. I'm sending it back for a replacement now. I did my utmost best and performed all the upgrades and installs, including confirming the issue with NVDA. But I'll need them to set up my account correctly to be able to test while I work. I'll ask if I can keep the next tablet long-term so we have a way to test for PC/touch/NVDA.

**Comment by geospatialem** [comment]
Reallocating to the May release milestone.
